### PR TITLE
Fix bugs S2 Super Resolution

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
- [x] Raise UP42Error if geometries are passed but clip_to_aoi is false;

- [x] Return a uint16 raster instead of float

### CODE REVIEW

**Comment by nekhtiari** [comment]
Nice,LGTM

**Review by thaisbendixen42 — APPROVED** [review]

**Review by chrieke — APPROVED** [review]

File: marketplace.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-"publicVersion": "2.2.0"
+"publicVersion": "2.2.1"
```
Comment (markusuwe) [inline marketplace.json:2]: This is more than a bug fix, as the data type of the output is changed. Would suggest to go to 2.3.0.

**Review by markusuwe — APPROVED** [review]
LGTM

File: marketplace.json (hunk 1 above)
Comment (rodrigoalmeida94) [reply marketplace.json:2]: Good point, updating!

**Pull request merged**